acquia_lift.admin.wizard.inc

```diff
@@ -840,6 +933,9 @@ function acquia_lift_personalize_campaign_wizard_submit(&$form, &$form_state) {
     $submit_function($form, $form_state, $agent_data, $agent_instance);
   }
 
+  if (isset($form_state['values']['variations']['variation_set_handling'])) {
+    $agent_data->data['variation_set_handling'] = $form_state['values']['variations']['variation_set_handling'];
+  }
```

Comment:
Since its the same agent instance getting passed to the steps, we can change things in the steps and then at the very end the save gets called in the main submit without having to call it over and over in each step.  It's how a few other changes already are handled - like the stuff in the review form and the agent advanced settings, etc.